build.gradle

```diff
@@ -124,9 +114,11 @@ subprojects {
     }
 
     dependencies {
+        implementation platform(group: 'org.springframework.boot', name: 'spring-boot-dependencies', version: versions.spring_boot)
```

Comment:
I'm not so sure about using spring-boot-dependencies here. envoy-control-runner is just a small and non-crucial part of envoy-control. By using spring-boot dependencies in other subprojects we make them dependent on Spring Boot.

Reply:
Only on its bill of materials, of course I can add them manually if that will be the chosen way.